lib/index.ts

```diff
@@ -546,6 +546,18 @@ export default class Backburner {
     this._ensureInstance();
   }
 
+  public getDebugInfo() {
+    if (this.DEBUG) {
+      return {
+        counters: this.counters,
```

Comment:
I'm not convinced we need this. @rwjblue - what are you thinking this would be good for?